devops/pypi-release.yml

```diff
@@ -59,7 +59,7 @@ stages:
 
   - template: notebook-job-template.yml
     parameters:
-      pyVersions: [3.6]
+      pyVersions: [3.5, 3.6]
```

Comment:
why no 3.7 here?

Reply:
Can't remember offhand. But I also have a task for getting the NB tests running on more platforms.